protostar/commands/init/project_creator/new_project_creator.py

```diff
@@ -49,8 +49,13 @@ def _gather_input(self) -> "NewProjectCreator.NewProjectConfig":
 
         return NewProjectCreator.NewProjectConfig(project_dirname)
 
-    def _create_project(self, user_input: "NewProjectCreator.NewProjectConfig") -> None:
+    def _create_project(
+        self, user_input: "NewProjectCreator.NewProjectConfig", is_cairo_1: bool
+    ) -> None:
         output_dir_path = self._output_dir_path
         project_root_path = output_dir_path / user_input.project_dirname
-        self.copy_template("default", project_root_path)
+        if is_cairo_1:
+            self.copy_template("cairo1", project_root_path)
+        else:
+            self.copy_template("cairo0", project_root_path)
```

Comment:
I know it looks ugly, but it has to be done this way - this is an argument of type Literal["cairo0", "cairo1"]
https://peps.python.org/pep-0586/#backwards-compatibility